addons/web/static/src/legacy/scss/utils.scss

```diff
@@ -221,10 +221,14 @@
     @return $-with-support-font;
```

Comment:
Commit message:
>  This, combined with the above issue, made it impossible to set
the buttons primary style to "Fill".

Nitpicking but I think this is not accurate, choosing a value (e.g. setting to "Fill") is buggy in the same way with or without the bug in the theme enark definition.